fe/fe-core/src/main/java/org/apache/doris/analysis/ColumnDef.java

```diff
@@ -157,7 +159,7 @@ public void analyze(boolean isOlap) throws AnalysisException {
             final ScalarType targetType = (ScalarType) typeDef.getType();
```

Comment:
default len of StringType is MAX_STRING_LENGTH, it seems your change is the same as just remove codes between line 158 to line 163.

Reply:
> default len of StringType is MAX_STRING_LENGTH, it seems your change is the same as just remove codes between line 158 to line 163.

This is because I don't know where the length of string is set to 1, so I directly specify the maximum value here.

Reply:
This code sets the length of string type without length to 1, which should be consistent with the maximum length.